src/lib/utils.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * https://github.com/ShabadOS/desktop/blob/dev/app/frontend/src/lib/utils.js
```

Comment:
lol maybe we should bundle this into gurmukhi-utils or some shared lib

Reply:
Agreed

Reply:
https://github.com/ShabadOS/gurmukhi-utils/issues/136